test/Project.toml

```diff
@@ -0,0 +1,21 @@
+[deps]
```

Comment:
there shouldn't be a Project.toml here.

Reply:
My understanding is that this is the new way to handle the tests in pkg.

Reply:
for now, do it the way that's standard across the organization. We can change everything uniformly if that's what the Pkg people want. But I haven't heard anything at least.

Reply:
Fixed via a2a3572